src/gsi/intpm2_5.f90

```diff
@@ -226,7 +227,7 @@ subroutine intpm2_5_(pm2_5head,rval,sval)
                w7* spm2_5(j7)+w8* spm2_5(j8)
           val=val *pm2_5ptr%pm25wc(imodes_cmaq_fv3(iaero))
           nullify(spm2_5)
-!
+
```

Comment:
nitpick:  What's this space for?

Reply:
Deleted a comment line and added a blank line which works as a separation from the next pieces of code.